pftools/python/parflow/tools/builders.py

```diff
@@ -583,7 +583,7 @@ def load_default_properties(self, database=None):
 
         return self
 
-    def apply(self, run=None, name_registration=True):
+    def apply(self, run=None, name_registration=True, infer_key_names=False):
```

Comment:
I've made it so that you need to turn on the code that reads names from the table because I am not 100% confident in it yet